addon/components/frost-action-bar.js

```diff
@@ -31,6 +32,12 @@ export default Component.extend({
     i18n: PropTypes.shape({
       formatSelectedItemsLabel: PropTypes.func.isRequired
     }),
+    moreActionButtons: PropTypes.oneOfType([
+      PropTypes.arrayOf(PropTypes.object),
```

Comment:
PropTypes.arrayOf(PropTypes.EmberComponent)

Reply:
This would be in place of, not in addition to.  The _README.md_ changes say
> The `moreActionButtons` property only accepts `frost-buttons`.

Reply:
I accepts either frost-buttons or plain objects. This is why I used PropTypes.object because specifying both here only really serves as documentation.